core/src/main/resources/defaults/clustertemplates/azure/azure-opdb-710.json

```diff
@@ -0,0 +1,122 @@
+{
+  "name": "Operational Database with Phoenix for AWS",
```

Comment:
nit: with Apache Phoenix

_maybe_ pedantic, but I'm worried where cloudbreak might display this name and we then get into a trademarks issue with the ASF.

Reply:
This is displayed very prominently, but your suggestion definitely won't fit into the list.

How about replacing "with Phoenix" with "with SQL" in the descriptions ? That way we fit into a UI, and sidestep the trademark problem. The other DataHbus do similarly, they do not name the specific software packages.

Reply:
I like that suggestion!

I think this is all temporary, too, and the normal will be to have SQL in all opdb :)

Reply:
Thanks. It's done. Could you resolve/remove the request for change to indicate that this is resolved ?